src/reset_reason.rs

```diff
@@ -48,12 +30,12 @@ impl TryFrom<esp_reset_reason_t> for ResetReason {
             esp_reset_reason_t_ESP_RST_BROWNOUT => ResetReason::Brownout,
             esp_reset_reason_t_ESP_RST_TASK_WDT => ResetReason::TaskWatchdog,
             esp_reset_reason_t_ESP_RST_DEEPSLEEP => ResetReason::DeepSleep,
-            _ => return Err(InvalidResetReason(value)),
-        })
+            _ => panic!(),
```

Comment:
`unreachable!()`?

Reply:
done in https://github.com/esp-rs/esp-idf-hal/pull/130/commits/b57bbc1575446fdc9521fc7320be9ce695c5b0a8